android/jni/app-android.cpp

```diff
@@ -29,7 +36,351 @@
 
 #include "app-android.h"
 
-static JNIEnv *jniEnvUI;
+// HACK: Use NativeQueryConfig
```

Comment:
Hmm.  Is it really awful to use g_Config from here?  I guess it depends on how generic we want to keep it.

-[Unknown]